src/app/file-browser/components/download-button/download-button.component.scss

```diff
@@ -0,0 +1,51 @@
+@mixin flex {
+  display: flex;
+  align-items: center;
+}
+
+
+.download-container{
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid black;
+  border-radius: 4px;
+  padding-bottom: 6px;
+
+  .download-icon-text{
+    display: flex;
+    align-items: center;
+    @include flex();
+    &>span{
+      margin-left: 8px;
+    }
+
+  }
+  .download-icon{
+    @include flex();
+   
+  }
+}
+
+/* @format */
```

Comment:
```suggestion
/* @format */
@mixin flex {
  display: flex;
  align-items: center;
}


.download-container{
  cursor: pointer;
  display: flex;
  justify-content: space-between;
  border-bottom: 1px solid black;
  border-radius: 4px;
  padding-bottom: 6px;

  .download-icon-text{
    display: flex;
    align-items: center;
    @include flex();
    &>span{
      margin-left: 8px;
    }

  }
  .download-icon{
    @include flex();
   
  }
}

```
Minor nitpick: can you run prettier on this file and move the `@format` pragma to the top?